File: app/src/main/java/com/example/trabalhosemestral/Entity/Material.java
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "material")
data class Material(
    @PrimaryKey(autoGenerate = true)  // Gera automaticamente um ID único para cada usuário
    val id: Int = 0,
    val nome: String,
    val marca: String,
    val quantidade: String,
)

File: app/src/main/java/com/example/trabalhosemestral/entity/Material.kt
package com.example.trabalhosemestral.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "materiais")
open class Material(
    @PrimaryKey(autoGenerate = true) val id: Int = 0,
    val nome: String,
    val marca: String,
    val quantidade: Int,
    
)